### ISSUE TITLE
🔗 🐞 Bug Report: Stop displaying .elementor-empty-view when not needed (Duplicate of #22406)

### ISSUE THREAD

**Issue by ofmarconi** [elementor/elementor#22545]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

![image](https://github.com/elementor/elementor/assets/6594795/1ee38eb1-a0ab-492c-8385-64f6a17d5ae2)

It needs to be hidden, it doesn't make sense if I marked 1 line and the Grid Outline is off.

It just gets in the way of viewing in editor mode.

### Steps to reproduce

DISPLAY GRID

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: Ubuntu 20.04 v10.3.38
	PHP Version: 7.4.33
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2

	WP Multisite: No
	Max Upload Size: 100 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: pt-BR
	Timezone: -3

	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.7.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: pt_BR
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.13.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.2
		Author: Elementor.com

	Jet Appointments Booking
		Version: 2.0.2
		Author: Crocoblock

	JetElements For Elementor
		Version: 2.6.9
		Author: Crocoblock

	JetEngine
		Version: 3.1.5
		Author: Crocoblock

	JetEngine - query results count plain number
		Version: 1.0.0
		Author: Crocoblock

	JetPopup
		Version: 2.0.0
		Author: Crocoblock

	JetSearch
		Version: 3.0.3
		Author: Crocoblock

	JetSmartFilters
		Version: 3.1.1
		Author: Crocoblock

	JetTricks
		Version: 1.4.4
		Author: Crocoblock

	LiteSpeed Cache
		Version: 5.4
		Author: LiteSpeed Technologies

	ManageWP - Worker
		Version: 4.9.17
		Author: GoDaddy

	Performance Lab
		Version: 2.3.0
		Author: WordPress Performance Team

	Rank Math SEO
		Version: 1.0.115
		Author: Rank Math

	Retarget Marks
		Version: 2.0
		Author: Marconi

	Uncanny Automator
		Version: 4.15
		Author: Uncanny Automator, Uncanny Owl

	WP Offload SES Lite
		Version: 1.6.4
		Author: Delicious Brains

	WPS Hide Login
		Version: 1.9.8
		Author: WPServeur, NicolasKulka, wpformation

	Yoast Duplicate Post
		Version: 4.5
		Author: Enrico Battocchi & Team Yoast


== Must-Use Plugins ==
	ManageWP - Worker Loader
		Version: 1.0.0
		Author: GoDaddy


== Recursos ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrações ==
	
	recaptcha_v3: Ativo


== Experimentos do Elementor ==
	Saída DOM otimizada: Ativo
	Carregamento melhorado de recursos: Ativo
	Carregamento melhorado de CSS: Ativo
	Ícones da fonte inline: Inativo por padrão
	Melhorias na acessibilidade: Ativo por padrão
	Pontos de quebra adicionais personalizados: Inativo
	admin_menu_rearrangement: Inativo por padrão
	Flexbox Container: Ativo
	Atualize a biblioteca Swiper: Ativo
	Grid Container: Ativo
	Padrão para Novo Construtor de Tema: Inativo
	Tema Hello - Cabeçalho & Rodapé: Inativo por padrão
	Editor Top Bar: Inativo por padrão
	Landing Pages: Inativo
	Elementos aninhados: Ativo
	Imagens com carregamento gradual: Inativo por padrão
	Transições da página: Ativo por padrão
	Atividades Desempenhadas: Inativo
	Repetir: Ativo por padrão
	Envios do formulário: Ativo por padrão
	Pressão da Rolagem: Inativo
	Menu: Ativo


== Registo ==
	


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatível
	JetElements For Elementor: Incompatível

== Elementor Pro - Compatibility Tag ==
	
	JetElements For Elementor: Incompatível
				
````

</details>

**Comment by ofmarconi** [comment]
```
.e-grid > .e-con-inner > .elementor-empty-view {
    display: none;
}
```

**Comment by nicholaszein** [comment]
Hello @ofmarconi!

Thank you for your submission! 🙏 

### 🔗 🐞 This issue is a Duplicate of #22406 22406

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Best regards,

Nick

**Comment by nicholaszein** [comment]
Hello there @ofmarconi!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.16! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
